### PR TITLE
bpo-42547: fix add_argument(metavar=<tuple>, nargs='+') for positional arguments

### PR DESCRIPTION
**Pull request by m-khvoinitsky**
https://bugs.python.org/issue42547

* Issue: gh-86713

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).



## CLA Missing

Our records indicate the following people have not signed the CLA:

@m-khvoinitsky

For legal reasons we need all the people listed to sign the CLA before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: Lib/argparse.py [hunk 1]
```diff
@@ -547,7 +547,7 @@ def _format_action(self, action):
     def _format_action_invocation(self, action):
         if not action.option_strings:
             default = self._get_default_metavar_for_positional(action)
-            metavar, = self._metavar_formatter(action, default)(1)
+            metavar = self._metavar_formatter(action, default)(1)[0]
```
Comment (ZackerySpytz) [inline Lib/argparse.py:550]: Sorry, but there really needs to be a unit test for this change.

File: Misc/NEWS.d/next/Library/2020-12-03-00-40-12.bpo-42547.O2BJhp.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+argparse: fixed `add_argument(metavar=<tuple>, nargs='+')` for positional arguments
```
Comment (ZackerySpytz) [inline Misc/NEWS.d/next/Library/2020-12-03-00-40-12.bpo-42547.O2BJhp.rst:1]: Please add more details and use full sentences. Code snippets should be enclosed in double backquotes.

Please replace "argparse" with
```rst
:mod:`argparse`
```

**Comment by sys-ptipu** [comment]
**:closed_book: MEV SW OD Build <a href=https://cje-ir-prod01.devtools.intel.com/nex-ncng-ipu/job/MEV/job/run_od_builds/27839/display/redirect>#27839</a> result: FAILURE**
 | Manifest Name | Build Status | Jenkins Link | Artifacts | Unsuccessful Stages | Description |
 | - | - | - | - | - | - |
 | mev-ts-trunk  | FAILURE  | <a href=https://cje-ir-prod01.devtools.intel.com/nex-ncng-ipu/job/MEV/job/builds/job/official/job/mev-ts-trunk/59604/display/redirect>59604</a>  | <a href=https://ubit-artifactory-or.intel.com/artifactory/list/mountevans_sw_bsp-or-local/builds/official/mev-ts-trunk/od/mev-ts-trunk-od-59604> Artifacts </a> | <a href=https://cje-ir-prod01.devtools.intel.com/nex-ncng-ipu/job/MEV/job/builds/job/official/job/mev-ts-trunk/59604/display/redirect>IPU SDK Build</a> |  |
 | mev-trunk  | FAILURE  | <a href=https://cje-ir-prod01.devtools.intel.com/nex-ncng-ipu/job/MEV/job/builds/job/official/job/mev-trunk/56680/display/redirect>56680</a>  | <a href=https://ubit-artifactory-or.intel.com/artifactory/list/mountevans_sw_bsp-or-local/builds/official/mev-trunk/od/mev-trunk-od-56680> Artifacts </a> | <a href=https://cje-ir-prod01.devtools.intel.com/nex-ncng-ipu/job/MEV/job/builds/job/official/job/mev-trunk/56680/display/redirect>Simics_C0_ACC</a> |  |
 | mev-release | NOT AFFECTED | | | |
 | mev-ts-release | NOT AFFECTED | | | |
 | mev-ts-release-1.6.1 | NOT AFFECTED | | | |
 | mev-ts-release-bitbake | NOT AFFECTED | | | |
 | mev-ts-release-external | NOT AFFECTED | | | |
 | mev-ts-trunk-bitbake | NOT AFFECTED | | | |
 | mev-ts-trunk-bitbake-c1 | NOT AFFECTED | | | |
 | mev-ts-trunk-external | NOT AFFECTED | | | |
 | mev-upstream-idpf | NOT AFFECTED | | | |

**Comment by sys-ptipu** [comment]
**:closed_book: MEV SW OD Build <a href=https://cje-ir-prod01.devtools.intel.com/nex-ncng-ipu/job/MEV/job/run_od_builds/27894/display/redirect>#27894</a> result: FAILURE**
 | Manifest Name | Build Status | Jenkins Link | Artifacts | Unsuccessful Stages | Description |
 | - | - | - | - | - | - |
 | mev-trunk  | FAILURE  | <a href=https://cje-ir-prod01.devtools.intel.com/nex-ncng-ipu/job/MEV/job/builds/job/official/job/mev-trunk/56702/display/redirect>56702</a>  | <a href=https://ubit-artifactory-or.intel.com/artifactory/list/mountevans_sw_bsp-or-local/builds/official/mev-trunk/od/mev-trunk-od-56702> Artifacts </a> | <a href=https://cje-ir-prod01.devtools.intel.com/nex-ncng-ipu/job/MEV/job/builds/job/official/job/mev-trunk/56702/display/redirect>IPU_IMC_6_6_4</a> |  |
 | mev-ts-trunk  | SUCCESS  | <a href=https://cje-ir-prod01.devtools.intel.com/nex-ncng-ipu/job/MEV/job/builds/job/official/job/mev-ts-trunk/59633/display/redirect>59633</a>  | <a href=https://ubit-artifactory-or.intel.com/artifactory/list/mountevans_sw_bsp-or-local/builds/official/mev-ts-trunk/od/mev-ts-trunk-od-59633> Artifacts </a> |  |  |
 | mev-release | NOT AFFECTED | | | |
 | mev-ts-release | NOT AFFECTED | | | |
 | mev-ts-release-1.6.1 | NOT AFFECTED | | | |
 | mev-ts-release-bitbake | NOT AFFECTED | | | |
 | mev-ts-release-external | NOT AFFECTED | | | |
 | mev-ts-trunk-bitbake | NOT AFFECTED | | | |
 | mev-ts-trunk-bitbake-c1 | NOT AFFECTED | | | |
 | mev-ts-trunk-external | NOT AFFECTED | | | |
 | mev-upstream-idpf | NOT AFFECTED | | | |

**Comment by sys-ptipu** [comment]
mev-ts-trunk BAT tests: [success](http://10.166.204.21/test_session?id=59188) - build 59633 - it is required if PR have BAT MEV status check as mandatory. If failed, please review Tasks in provided link to Berta. For questions please contact with ipu.ci.triage.team@intel.com

**Comment by serhiy-storchaka** [comment]
Thank you for your PR, @m-khvoinitsky, but it was not be merged dues to lack of tests. And tests could show that these changes are not enough.

#124782 is a more complete solution for this issue.

**Pull request closed without merging**